### ISSUE TITLE
Drag and drop difference between Firefox and Chromium browsers

### ISSUE THREAD

**Issue by liviuconcioiu** [phpmyadmin/phpmyadmin#18557]
### Describe the bug

The drag and drop is activated in Chromium browsers, when an icon from phpMyAdmin is dragged accidentally. This doesn't happen on Firefox.

### To Reproduce

1. Go to https://demo.phpmyadmin.net/QA_5_2/index.php?route=/
2. Drag the phpMyAdmin logo
3. See difference between Firefox and Chrome (it gets activated while it shouldn't)

### Expected behavior

Drag and drop should work like in Firefox - it doesn't get activated accidentally with icons from phpMyAdmin.

### Screenshots

https://github.com/phpmyadmin/phpmyadmin/assets/25424343/859b67aa-9e93-4d4a-9e48-01b837ce3fbe

### Server configuration

- phpMyAdmin version: 5.2.2-dev, 6.0.0-dev

### Client configuration

- Browser: Chrome

**Comment by williamdes** [comment]
I can confirm this bug, it's a real pain.

For contributors: please create a new branch from QA_5_2 and submit a PR to the target QA_5_2.

**Comment by RajSinha67** [comment]
Do we need to disable it to all browser ? If so, we can make '**is_drag_drop_import_enabled**' condition false. I have checked it by disabling this variable and it is supposed to solve our bug.

**Comment by ibennetch** [comment]
> Do we need to disable it in all browser ?

No,unfortunately it's not so simple. This report is about some browsers incorrectly thinking a user is trying to drag and drop when they aren't.

**Comment by RajSinha67** [comment]
I guess, its something has to do with 'dataTransfer.types' of the drag event as the condition is meeting for firefox but not with chrome.
It seems like for drag operation different files are getting included for different browsers and there is a condition checking based on which (drag & drop screen) is activated or de-activated.

**Comment by RajSinha67** [comment]
I guess we should check on more number of browsers to confirm whether or not this is browser dependent bug or not. Any suggestions ?

**Comment by williamdes** [comment]
> I guess we should check on more number of browsers to confirm whether this is browser dependent bug or not. Any suggestions ?

Use [BrowserStack live](https://www.browserstack.com/live) and you can try different browsers online

**Comment by harshitrajsinha** [comment]
For browsers like - Chrome, Edge, Brave, Opera => Drag drop is active (Why? MIME type = 'File' is present)
For browsers like - Firefox, Falkon, Midori => Drag drop is de-active (Why? MIME type = 'File' is absent)

It seems the issue is due to MIME type inconsistency among browsers. I found couple of stackoverflow discussions also regarding similar issue faced by others.

MIME type are like labels that indicate a software about the format or nature of data being dragged or dropped.

**Comment by harshitrajsinha** [comment]
Also, I found a comment in the codebase regarding the drag event

>  // We don't want to prevent users from using
>  // browser's default drag-drop feature on some page(s)
